main.py

```diff
@@ -56,28 +56,30 @@ def home():
 def register():
     content = request.json
     if content["id"] in users.keys():
-        return '{"error": "ID already registered"}'
+        return jsonify('{"error": "ID already registered"}')
     jwt_data = {"user": content["id"]}
 
     encoded_jwt = jwt.encode(jwt_data, secret_key, algorithm="HS256")
     print("JWT generated - " + encoded_jwt)
     user1 = user.User(content["id"], content["name"], encoded_jwt)
     users[content["id"]] = user1
-    return json.dumps(user1.__dict__)
+
+    return jsonify(user1.toJson())
 
 
 @app.route("/add_note", methods=["POST"])
 @token_required
 def add_note(current_user):
     print("current user - " + json.dumps(current_user.__dict__))
     content = request.json
+    print(content)
```

Comment:
nit: remove or comment out this print statement

Reply:
not touching main code debug at this point